rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidConstants.java

```diff
@@ -78,6 +78,7 @@
         String PASSPORT_NO = "passport_no";
         String LAT = "lat";
         String LNG = "lng";
+        String PATIENT_DETAIL = "patient_detail";
```

Comment:
Please make this `patient_details` instead.

Reply:
PATIENT_DETAIL key removed